src/controllers/observation-controller.ts

```diff
@@ -54,11 +56,38 @@ export default class ObservationController extends AbstractController {
 
   private getObservations = async (req: RequestWithPageParams, res: Response, next: NextFunction): Promise<void> => {
     try {
+      const { lang = 'desc_eng' }: { lang: Locale } = req.query;
```

Comment:
please, dont use such value like `desc_#{lang}`, evenmore it's uncosistent with existing endpoints.